tests/e2e/distribution.go

```diff
@@ -75,3 +77,120 @@ func (s *CCVTestSuite) TestRewardsDistribution() {
 	s.Require().Greater(ibcCoinIndex, -1)
 	s.Require().True(communityCoins[ibcCoinIndex].Amount.Equal(sdk.NewDecCoinFromCoin(providerExpectedRewards[0]).Amount))
 }
+
+// TestEndBlockRD tests that the last transmission block height (LTBH) is correctly updated after the expected
+// number of block have passed. It also checks that the IBC transfer transfer states are discarded if
+// the reward distribution to the provider has failed.
+func (s *CCVTestSuite) TestEndBlockRD() {
```

Comment:
After working on this test more, it seems like a UT test for `EndBlockRD`, and how that updates the LTBH. Is there a test that validates the new (higher level) behavior described in the PR description? 

> the transfer is attempted again after BlocksPerDistributionTransmission blocks pass